openwisp_users/tests/test_admin.py

```diff
@@ -124,11 +115,12 @@ def test_admin_change_user_is_superuser_editable(self):
         html = '<input type="checkbox" name="is_superuser"'
         self.assertContains(response, html)
 
-    def test_admin_change_user_is_superuser_readonly(self):
+    def test_admin_change_user_is_superuser_absent(self):
         operator = self._create_operator()
+        self._create_org_user()
         self.client.force_login(operator)
         response = self.client.get(reverse('admin:openwisp_users_user_change', args=[operator.pk]))
-        html = '<input type="checkbox" name="is_superuser" checked id="id_is_superuser">'
+        html = 'name="is_superuser"'
```

Comment:
why did you have to change this?

Reply:
True, it wouldn't make a difference, i'll change it back! :sweat_smile: